xml/System.Data/IsolationLevel.xml

```diff
@@ -37,13 +37,14 @@
     <remarks>
       <format type="text/markdown"><![CDATA[  
   
-## Remarks  
- The <xref:System.Data.IsolationLevel> values are used by a .NET Framework data provider when performing a transaction.  
-  
- The <xref:System.Data.IsolationLevel> remains in effect until explicitly changed, but it can be changed at any time. The new value is used at execution time, not parse time. If changed during a transaction, the expected behavior of the server is to apply the new locking level to all statements remaining.  
-  
-   
-  
+## Remarks
+
+The <xref:System.Data.IsolationLevel> values are used by a .NET Framework data provider when performing a transaction.
+
+The <xref:System.Data.IsolationLevel> remains in effect until explicitly changed, but it can be changed at any time. The new value is used at execution time, not parse time. If changed during a transaction, the expected behavior of the server is to apply the new locking level to all statements remaining.  
```

Comment:
```suggestion
The `IsolationLevel` remains in effect until explicitly changed, but it can be changed at any time. The new value is used at execution time, not parse time. If changed during a transaction, the expected behavior of the server is to apply the new locking level to all statements remaining.  
```